test/test_prelude.py

```diff
@@ -0,0 +1,182 @@
+# coding: utf-8
+
+from pathlib import Path
+import shutil
+import os
+import glob
+import nibabel as nib
+import numpy as np
+import subprocess
+
+from shimmingtoolbox.unwrap import prelude
+
+
+class TestCore(object):
+
+    def setup(self):
+        # Get the directory where this current file is saved
+        self.full_path = Path(__file__).resolve().parent
+        print(self.full_path)
+        # "test/" directory
+        self.test_path = self.full_path
+
+        # Create temp folder
+        # Get the directory where this current file is saved
+        self.tmp_path = self.test_path / '__tmp__'
+        if not self.tmp_path.exists():
+            self.tmp_path.mkdir()
+        print(self.tmp_path)
+        self.toolbox_path = self.test_path.parent
+
+        self.get_phases_mags_affines()
+
+    def teardown(self):
```

Comment:
We could check to integrate the unittest library. It helps to create unit test and includes classes, tear down functions, mock classes and setup functions. It's another library to learn, but it is quite powerfull.

Reply:
i'd rather stick with pytest, just because the rest of the lab is using it (when new students come in, they get help from other students)

Reply:
Of course pytest is also an option. I was just saying this because I didn't see the required import.